rust_src/src/data.rs

```diff
@@ -844,7 +844,7 @@ pub fn string_to_number_lisp(mut string: LispStringRef, base: Option<EmacsInt>)
         }
     }
 
-    match unsafe { string_to_number(p, b as i32, true) } {
+    match unsafe { string_to_number(p, b as i32, 0) } {
```

Comment:
`S2N_IGNORE_TRAILING` in lisp.h, which is 1 in these code, seems to be an alternative for `true`.

Reply:
I just looked at the emacs source and it is 0. do you see something different?

Reply:
I'm seeing this. I was wrong, I should suggest `I2N_IGNORE_TRAILING | S2N_OVERFLOW_TO_NUMBER`.

https://github.com/remacs/remacs/blob/7bedc8812bd7ca1d9cf36636322068b28b690a85/src/data.c#L2737-L2738